controllers/topics.js

```diff
@@ -0,0 +1,11 @@
+const db = require('../db/connection')
+const { fetchTopics } = require('../models/topics')
+
+exports.getTopics = (req, res, next) => {
+    fetchTopics().then((topics) => {
+        res.status(200).send({ topics })
+    })
+    .catch((err) => {
+        throw new Error(err)
```

Comment:
instead of throwing a new error, we have something passed into the function that we're not using to pass the error on...